packages/alea-frontend/components/QuizDashboard.tsx

```diff
@@ -336,22 +337,34 @@ const QuizDashboard: NextPage<QuizDashboardProps> = ({ courseId, quizId, onQuizI
         </Select>
       </FormControl>
       {accessType == 'MUTATE' && (
-        <QuizFileReader setCss={setCss} setTitle={setTitle} setProblems={setProblems} />
+        <QuizFileReader
+          setCss={setCss}
+          setTitle={setTitle}
+          setProblems={setProblems}
+          setErrorMessage={setUploadErrorMessage}
+        />
       )}
       <br />
       <i>{Object.keys(problems).length} problems found.</i>
       <br />
 
       {selectedQuiz && (
-        <Typography sx={{ color: 'red' }} component="span" fontWeight="bold">
+        <Typography sx={{ color: 'red'  }} component="span" fontWeight="bold">
           {formErrorReason}
         </Typography>
       )}
+
+      {uploadErrorMessage && (
+        <Typography sx={{ color: 'red' }} fontWeight="bold">
+          {uploadErrorMessage}
+        </Typography>
+      )}
```

Comment:
Check directly if the variable `css` (from `const [css, setCss] = useState<FTML.CSS[]>([]);`) is empty.
In general, always try to avoid program logic based on error messages.  

```suggestion
      {!css?.length && problems.length > 0 && (
        <Typography sx={{ color: 'red' }} fontWeight="bold">
          CSS content is missing. Please try refreshing and re-uploading the quiz file.
        </Typography>
      )}
```